hybrid/pv_source.py

```diff
@@ -35,10 +35,15 @@ def __init__(self,
 
         self.dc_degradation = [0]
 
-        params: Optional[PVGridParameters] = None
-        if 'layout_params' in pv_config.keys():
-            params: PVGridParameters = pv_config['layout_params']
-        self._layout = PVLayout(site, system_model, params)
+        if 'layout_model' in pv_config.keys():
```

Comment:
Can this `layout_model` key-value pair be documented in this file?

Reply:
Done